pkg/envfuncs/third_party_funcs.go

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
This file can be moved under third-party/flux to maintain the structuring we followed for the other third-party support.

Reply:
I second that. Putting flux references in pkg would make it a required dependency when pulling the project, whether you want it or not.